### ISSUE TITLE
Enabling Secure Boot breaks Monterey booting on Lenovo Yoga 730 - OC 0.7.6

### ISSUE THREAD

**Issue by arisingh8** [acidanthera/bugtracker#1915]
Laptop model - Lenovo Yoga 730-15IKB
CPU - i7-8550u Kaby Lake-R
iGPU - Intel UHD 620
dGPU (not used) - Nvidia GTX 1050 mobile
RAM - 16GB DDR4 (assumed 2400 mhz)
Storage - ADATA SX8200 Pro 500 GB
Audio Codec - Realtek ALC236
Wifi/Bluetooth - Intel Wireless-AC 7265 (with AirportItlwm)
Keyboard is PS2, trackpad/touchscreen are I2C

I created a new EFI based on OpenCore 0.7.6 and macOS 12 for my Lenovo Yoga 730 and was able to boot into Recovery off of my USB. I started the install process, but started getting errors after the first restart when the PC tried booting into the macOS installer. The errors looked Apple Secure Boot related, so I set `SecureBootModel` to `Disabled` then was able to install Monterey. 

Now, I am unable to boot into macOS with `SecureBootModel` set to `Default` or `x86legacy`, receiving a flurry of what looks like Apple Secure Boot issues then a final `OCB: StartImage failed - Aborted`. I am running with a `MacBookPro14,1` SMBIOS (non-T2). I can boot fine with `SecureBootModel` set to `Disabled`.

GitHub won't let me attach files, so here is a Google Drive link to my EFI and latest boot failure: https://drive.google.com/drive/folders/18qtIjBD0ZVcHY3N6HyFwlhdoOpPupneB?usp=sharing

**Comment by vit9696** [comment]
Wrong Booter quirks. Memory map allocation failure.

**Comment by arisingh8** [comment]
Why does it boot without Secure Boot then? 

I noticed the memory map errors earlier before the Secure Boot issue cropped up and thought that it was resolved by my MmioWhitelist+DevirtualizeMmio. Does Secure Boot affect memory map allocation?

**Comment by vit9696** [comment]
https://github.com/acidanthera/OpenCorePkg/blob/ea58c41/Docs/Configuration.tex#L4282

**Comment by vit9696** [comment]
I would say you can try ProtectUefiServices first, but most likely you need ProvideMaxSlide.

**Comment by arisingh8** [comment]
So I tried ProtectUefiServices first, which didn't change anything and still failed to boot with the same error. I then tried finding a value for ProvideMaxSlide by first trying slide=1 as a boot-arg, which failed to boot--so I do not have any method for finding a value. Disabling DevirtualizeMmio in an effort to get rid of the MMIO errors also fails to boot. 

I have labeled added the logs from these failures to the Google Drive folder I originally posted.
